src/rgw/rgw_rados.cc

```diff
@@ -80,6 +80,8 @@ using namespace librados;
 
 #include "compressor/Compressor.h"
 
+#include "fmt/format.h"
```

Comment:
could you move this `#include` before `#include "include/random.h"` and after `#include <map>`?